pom.xml

```diff
@@ -53,12 +53,16 @@
     <openbracket>{</openbracket>
     <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
     <aether.version>1.1.0</aether.version>
-    <maven.version>3.9.0</maven.version>
+    <maven.version>3.8.1</maven.version> <!-- if we update this all consumers need to be running at least this version -->
     <maven-plugin-tools.version>3.8.1</maven-plugin-tools.version> <!-- extra work needed when https://issues.apache.org/jira/browse/MPLUGIN-449 is fixed -->
     <plexus-containers.version>2.1.1</plexus-containers.version>
     <aether.version>1.1.0</aether.version>
   </properties>
 
+  <prerequisites>
```

Comment:
not strictly necessary - but as we do not know if we use an api that was only introduced in this version it prevents random blow ups in consumers.